website/content/docs/templates/hcl_templates/blocks/build/hcp_packer_registry.mdx

```diff
@@ -0,0 +1,47 @@
+---
+description: >
+  The hcp_packer_registry allows operators the ability to customize the metadata sent to HCP Packer Registry.
+  It configures the base details of an image that is created or updated within HCP PAR.
+page_title: hcp_packer_registry - build - Blocks
+---
+
+# The `hcp_packer_registry` block
+
+The `hcp_packer_registry` block allows operators the ability to customize the metadata sent to
+HCP Packer Registry. It configures the details of an image that is created or updated within HCP PAR.
+
+The presence of a `hcp_packer_registry` block will enable HCP PAR mode and all the builds within that build block
+will be pushed to the remote registry as far the appropriate HCP credentials are set (`HCP_CLIENT_ID` and `HCP_CLIENT_SECRET`).
```

Comment:
What do you think about adding in a little blurb about what will happen if the HCP credentials are not set?

```suggestion
will be pushed to the remote registry if the appropriate HCP credentials are set (`HCP_CLIENT_ID` and `HCP_CLIENT_SECRET`). If no HCP credentials are set Packer will fail the builds and exit immediately to avoid any potential artifact drift between the build providers and the Packer registry. 
```